working_agreement.md

```diff
@@ -28,14 +29,14 @@
 1. Building a diverse environment
     - Please be open to others perspectives, engage with them, and be willing to revise your own
     - We work to vocally support diverse perspectives
-    - To specifically amplify the work of Black, Indigenous, People-of-Color (BIPOC) researchers, we've committed to 
```

Comment:
Since this is more of actionable items, we can move this into the "Process" or a different section? 
For the values, it can just be "we will support and amplify the work of BIPOC researchers"